lib/pages/new_group/widget/expansion_participants_list.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:fluffychat/pages/contacts/presentation/model/presentation_contact.dart';
+import 'package:fluffychat/pages/new_private_chat/widget/expansion_contact_list_tile.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+class ExpansionParticipantsList extends StatefulWidget {
+
+  final Set<PresentationContact> contactsList;
+
+  const ExpansionParticipantsList({
+    super.key,
+    required this.contactsList,
+  });
+
+  @override
+  State<StatefulWidget> createState() => _ExpansionParticipantsListState();
+}
+
+class _ExpansionParticipantsListState extends State<ExpansionParticipantsList> {
+  bool isExpanded = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 0.0, right: 0.0),
```

Comment:
fyi, you can do `EdgeInsets.symmetric(horizontal: 0)` if you find it cleaner

Reply:
okey